kong/dao/schemas/plugins.lua

```diff
@@ -1,14 +1,31 @@
 local utils = require "kong.tools.utils"
 local Errors = require "kong.dao.errors"
 local db_errors = require "kong.db.errors"
+local singletons = require "kong.singletons"
+local constants = require "kong.constants"
 
 local function load_config_schema(plugin_t)
-  if plugin_t.name then
-    local loaded, plugin_schema = utils.load_module_if_exists("kong.plugins." .. plugin_t.name .. ".schema")
+  local plugin_name = plugin_t.name
+
+  if plugin_name then
+
+    if constants.DEPRECATED_PLUGINS[plugin_name] then
+      ngx.log(ngx.WARN, "the '" .. plugin_name .. "' plugin has been deprecated")
```

Comment:
We avoid concatenating strings on the Lua-land with `ngx.log`. Also we can improve the message:
```lua
ngx.log(ngx.WARN, "plugin '", plugin_name, "' has been deprecated")
```